catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
+which is denoted as "N/A".
 
 # creator
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The name of the creator of the media item. Some providers use "Unknown" or
+similar for unknown creators, see
+[issue #1326](https://github.com/WordPress/openverse/issues/1326).
 
 # creator_url
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The URL of the creator's page, usually on the source site.
 
 # title
 
 ## Description
 
-## Object Shape
+The title of the media item.
 
-## Selection Criteria
+## Existing Data Inconsistencies
 
-## Normalization and Validation
+Provider scripts may include html tags in record titles, see
+[issue #1441](https://github.com/WordPress/openverse/issues/1441)
 
-## Existing Data Inconsistencies
+Some Wikimedia titles in the database still include "FILE:" prefix, and
+unnecessary file extension, which is
+[hot-fixed](https://github.com/WordPress/openverse/blob/3329d307f5cdce0cbe1f1c13dbe17106970abdaa/frontend/src/utils/decode-media-data.ts#L50)
+in the frontend. Some titles were incorrectly decoded, for which there is a
+hot-fix in the frontend
 
 # meta_data
 
 ## Description
 
-## Object Shape
+A JSON object containing additional metadata about the media item. This must
+contain the `license_url` (automatically added by the `MediaStore` class from
+the `License` object).
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+Relevant information that is not covered by other fields should be added here.
+This includes such items as the dates of creation, publication, geographical
+data, descriptions, and popularity data.
 
 # tags
 
 ## Description
 
+The list of tags associated with the media item.
+
 ## Object Shape
 
-## Selection Criteria
+jsonb array of dictionaries: `{"name": "tag1", "provider": "wordpress"}`. Some
+tags are provider-generated, and include an `accuracy` field with a float value.
+
+If there are no tags, the field should be set to null, not an empty array or
+empty object.
 
 ## Normalization and Validation
 
-## Existing Data Inconsistencies
+Tags are cleaned in the `MediaStore` class. The tags that contain license slugs
+are removed because they are often misleading, using a different license than
+the media item itself (see
+https://github.com/cc-archive/cccatalog-api/issues/253).
 
-# watermarked
+## Existing Data Inconsistencies
 
-## Description
+The cleanup process in data refresh fixes the following tag inconsistencies:
 
-## Object Shape
+- Empty tags (`{}`) are filtered out, see
+  [cc-archive issue](https://github.com/cc-archive/cccatalog-api/issues/130)
+- Tags with license slugs are removed because they often don't match the media
+  item license. These tags, as well as some other deny-listed tags are filtered
```

Comment:
This looks like it's a duplicate of the above, should it just be referenced here? (with the URL as well?)